File: base-ui/src/app/models/product-cart.ts
import { Store } from "./store";

export class ProductCart {
    public productCode : string;
    public selectedDelivery : string;
    public store : Store;
    public quantity : number;
    public username : string;
    
    constructor(init? : Partial<ProductCart>){
        Object.assign(this, init);
    }
    
}

File: Base/Base/Models/ProductCart.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Base
{
    public class ProductCart
    {
        public string ProductCode { get; set; }
        public string SelectedDelivery { get; set; }
        public Store Store { get; set; }
        public int Quantity { get; set; }
        public string Username { get; set; }

        public void Show()
        {
            Console.WriteLine("ProductCode: " + ProductCode + ", ShippingDeliveryType : " + SelectedDelivery + ", Quantity: " + Quantity);
        }
    }
}
